Classes/Domain/Repository/AbleFindByUidList.php

```diff
@@ -4,6 +4,8 @@
 
 namespace Pixelant\PxaProductManager\Domain\Repository;
 
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
 use TYPO3\CMS\Extbase\Persistence\QueryResultInterface;
 
 trait AbleFindByUidList
```

Comment:
I suggest we refactor all of these `Able*` traits to proper `Can*`

```suggestion
trait CanFindByUidList
```